acceptance_tests/features/who.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const translation = require('./../../apps/complaints/translations/en/default');
+
+Feature('I am on the /who step');
+
+Scenario('The page loads and I am on the Personal and contact details page', (I) => {
```

Comment:
GIVEN a customer wants someone to make a complaint for them,
WHEN they access the UKVI complaint form,
THEN they will be asked to specify if they are "applicant" or "representative"
- we should check the label of the field

GIVEN I select Applicant
WHEN I click Save and Continue
THEN I progress to to the Applicant's Name page
we should add a blank applicant's name step and make sure continue links to it

GIVEN I select Representative
WHEN I click Save and Continue
THEN I progress to the Representative's Declaration page
we should make a blank declaration page and make sure it links to it